Classes/helpers.py

```diff
@@ -22,7 +22,7 @@ class RailPassLevel(Enum):
 """
 
 
-def invalid_string(string):
+def is_valid_string(string):
```

Comment:
These functions check for invalid data, so if we wanted to change them then "is_invalid_string" would make more sense